### PR TITLE
Show connection latency on stats card

### PR DESCRIPTION
**Pull request by xBelladonna**
- Shows the connection latency (ping time in milliseconds) as a field on the stats card when `pk;stats` is called

### CODE REVIEW

File: PluralKit.Bot/Commands/MiscCommands.cs [hunk 1]
```diff
@@ -43,10 +43,11 @@ public async Task Stats()
         {
             var messagesReceived = Metrics.Snapshot.GetForContext("Bot").Meters.First(m => m.MultidimensionalName == BotMetrics.MessagesReceived.Name).Value;
             var messagesProxied = Metrics.Snapshot.GetForContext("Bot").Meters.First(m => m.MultidimensionalName == BotMetrics.MessagesProxied.Name).Value;
-            
+
             var commandsRun = Metrics.Snapshot.GetForContext("Bot").Meters.First(m => m.MultidimensionalName == BotMetrics.CommandsRun.Name).Value;
-            
+
             await Context.Channel.SendMessageAsync(embed: new EmbedBuilder()
+                .AddField("Connection latency", $"{Context.Client.Latency}ms")
```
Comment (xSke) [inline PluralKit.Bot/Commands/MiscCommands.cs]: `DiscordShardedClient.Latency` returns average latency across all shards, perhaps it'd be more useful to display both that and the current shard's latency? (eg. `Client.GetShardFor(Context.Guild).Latency` - make sure to check if it's actually *in* a guild context)

Comment (xBelladonna) [reply PluralKit.Bot/Commands/MiscCommands.cs]: Resolved with 11bb41c

Comment (xBelladonna) [reply PluralKit.Bot/Commands/MiscCommands.cs]: Finalized with 8a74938

**Comment by xSke** [comment]
I'm going to retire the stats command and replace it with a `pk;ping` command soon:tm:, this isn't really needed then :/

**Pull request closed without merging**